header/src/main/resources/_header/i18n/index_de.properties

```diff
@@ -11,3 +11,6 @@ admin=Verwaltung
 users=Benutzer
 logout=Abmelden
 login=Anmelden
+remaining_days_msg_part1=Nur noch
+remaining_days_msg_part2=Tage zum
+remaining_days_msg_part3=Ändern Sie Ihr Passwort
```

Comment:
```suggestion
remaining_days_msg_part1=Nur noch
remaining_days_msg_part2=Tage um
remaining_days_msg_part3=Ihr Passwort zu ändern
```

Reply:
I need to refresh my german before being able to review this, but I do trust @tkohr anyway